Source/WebCore/Modules/fetch/FetchBodyOwner.cpp

```diff
@@ -138,47 +137,42 @@ void FetchBodyOwner::blob(Ref<DeferredPromise>&& promise)
         return;
     }
     m_isDisturbed = true;
-    m_body->blob(*this, WTFMove(promise), m_contentType);
+    m_body->blob(*this, WTFMove(promise));
 }
 
 void FetchBodyOwner::cloneBody(FetchBodyOwner& owner)
 {
     m_loadingError = owner.m_loadingError;
-
-    m_contentType = owner.m_contentType;
+    m_headers = FetchHeaders::create(owner.m_headers.get());
```

Comment:
Why is this needed?
We are only supposed to clone the body here, callers of cloneBody should clone the headers themselves?
This might be to handle FetchResponse::clone, in which case we might actually want to change FetchResponse::clone.

Reply:
no, thanks for catching this! I will move this logic to `FetchResponse::clone` and leave this function only take care of the clone of body 🙏 (we don't need to clone headers in `FetchRequest::clone` because it already pass the headers to `FetchRequest::create`)